test/jdk/ProblemList.txt

```diff
@@ -774,6 +774,10 @@ jdk/jfr/event/compiler/TestCodeSweeper.java                     8338127 generic-
 jdk/jfr/event/runtime/TestResidentSetSizeEvent.java             8309846 aix-ppc64
 jdk/jfr/jvm/TestWaste.java                                      8282427 generic-all
 
+java/lang/Thread/jni/AttachCurrentThread/AttachTest.java#id0    8343244 linux-aarch64,linux-ppc64le
+java/lang/Thread/jni/AttachCurrentThread/AttachTest.java#id1    8343244 linux-aarch64,linux-ppc64le
+jdk/jfr/event/gc/detailed/TestZPageAllocationEvent.java         8343244 linux-aarch64,linux-ppc64le
```

Comment:
Also IIUC the JFR test only failed with non-generational ZGC which has now been removed.